src/Network/Ethereum/Web3/Contract.purs

```diff
@@ -135,12 +136,19 @@ _call :: forall p a arep b brep e selector .
       => TransactionOptions
       -> ChainCursor
       -> Tagged (SProxy selector) a
-      -> Web3 p e b
+      -> Web3 p e (Either CallError b)
 _call txOptions cursor dat = do
-    let sel = toSelector <<< reflectSymbol $ (SProxy :: SProxy selector)
+    let sig = reflectSymbol $ (SProxy :: SProxy selector)
+        sel = toSelector sig
+        fullData = sel <> (genericABIEncode <<< untagged $ dat)
     res <- eth_call (txdata $ sel <> (genericABIEncode <<< untagged $ dat)) cursor
     case genericFromData res of
-        Nothing -> throwError <<< error $ "Unable to parse result"
-        Just x -> pure x
+      Left err ->
+        if res == mempty || res == nullWord
```

Comment:
I'm thinking this `nullWord` case can't happen, can anyone correct me @XertroV @cmditch